src/components/load3d/Load3DAnimationScene.vue

```diff
@@ -124,21 +124,24 @@ watch(
 watch(
   () => props.playing,
   (newValue) => {
-    load3DSceneRef.value.load3d.toggleAnimation(newValue)
+    // @ts-expect-error toogleAnimated not defined on type
```

Comment:
@jtydhr88 PTAL these issues. I think these watches are broken.

Reply:
as my test, it is still working well

Reply:
Nvm, I think here we need to narrow the type to `Load3DAnimation`.